infraohjelmointi_api/views/CoordinatorNoteViewSet.py

```diff
@@ -0,0 +1,78 @@
+from django.shortcuts import get_object_or_404
+
+from .BaseViewSet import BaseViewSet
+from infraohjelmointi_api.serializers.CoordinatorNoteSerializer import CoordinatorNoteSerializer
+from rest_framework.decorators import action
+from rest_framework.response import Response
+from rest_framework import status
+from infraohjelmointi_api.services.CoordinatorNoteService import (
+    CoordinatorNoteService,
+)
+
+class CoordinatorNoteViewSet(BaseViewSet):
+    """
+    API endpoint that allows coordinator notes to be viewed and added.
+    """
+
+    serializer_class = CoordinatorNoteSerializer
+
+    @action(
```

Comment:
No need to create this custom action to get all CoordinatorNotes. By default there is a method in place from the BaseViewSet class which allows all instances to be retrieved from the default endpoint. 
Visiting `/coordinator-notes/` should give all the notes without this action by default.